jetty-server/src/main/java/org/eclipse/jetty/server/HttpChannelOverHttp.java

```diff
@@ -75,6 +78,55 @@ public void abort(Throwable failure)
         _httpConnection.getGenerator().setPersistent(false);
     }
 
+    @Override
+    public boolean needContent()
+    {
+        if (_specialContent != null || _content != null)
```

Comment:
Need a comment somewhere saying why atomics/volatiles/locks are not needed in this class

Reply:
Done.